diff -dur mapserver-utils.orig//landuse.map mapserver-utils/landuse.map
--- mapserver-utils.orig//landuse.map	2011-05-30 12:20:08.000000000 +0200
+++ mapserver-utils/landuse.map	2011-05-30 12:46:21.000000000 +0200
@@ -251,7 +251,7 @@
     GROUP "default"
     CONNECTIONTYPE POSTGIS
     CONNECTION _db_connection
-    DATA 'way from (select way,waterway,osm_id from OSM_PREFIX_line where waterway IN ('river', 'stream', 'canal')) as foo using unique osm_id using srid=OSM_SRID'
+    DATA "way from (select way,waterway,osm_id from OSM_PREFIX_line where waterway IN ('river', 'stream', 'canal')) as foo using unique osm_id using srid=OSM_SRID"
     CLASSITEM "waterway"
     SIZEUNITS meters
     MAXSCALEDENOM 1500000
Only in mapserver-utils: osm-mapserver.map
diff -dur mapserver-utils.orig//postprocess.sql.in mapserver-utils/postprocess.sql.in
--- mapserver-utils.orig//postprocess.sql.in	2011-05-30 12:20:08.000000000 +0200
+++ mapserver-utils/postprocess.sql.in	2011-05-30 14:24:37.000000000 +0200
@@ -1,33 +1,33 @@
 -- optional, makes it easier to wrap place names
-update osm_point set name = replace(name,'-',' ');
+update OSM_PREFIX_point set name = replace(name,'-',' ');
 
 -- grant our user access to the data
-grant SELECT on osm_line to "openstreetmap";
-grant SELECT on osm_point to "openstreetmap";
-grant SELECT on osm_polygon to "openstreetmap";
+grant SELECT on OSM_PREFIX_line to "openstreetmap";
+grant SELECT on OSM_PREFIX_point to "openstreetmap";
+grant SELECT on OSM_PREFIX_polygon to "openstreetmap";
 
 -- correct a few common typos
-update osm_line set tunnel='yes' where tunnel='true';
-update osm_line set tunnel='no' where tunnel='false';
-update osm_line set tunnel='yes' where tunnel='yel';
-update osm_line set bridge='yes' where bridge='true';
+update OSM_PREFIX_line set tunnel='yes' where tunnel='true';
+update OSM_PREFIX_line set tunnel='no' where tunnel='false';
+update OSM_PREFIX_line set tunnel='yes' where tunnel='yel';
+update OSM_PREFIX_line set bridge='yes' where bridge='true';
 
-update osm_line set oneway='yes' where oneway='Yes';
-update osm_line set oneway='yes' where oneway='true';
-update osm_line set oneway='yes' where oneway='1';
+update OSM_PREFIX_line set oneway='yes' where oneway='Yes';
+update OSM_PREFIX_line set oneway='yes' where oneway='true';
+update OSM_PREFIX_line set oneway='yes' where oneway='1';
 
 -- create a few indexes on the data for speedier access
-create index osm_polygon_building_idx on osm_polygon(building);
-create index osm_polygon_amenity_idx on osm_polygon(amenity);
-create index osm_polygon_landuse_idx on osm_polygon(landuse);
-create index osm_line_highway_idx on osm_line(highway);
-create index osm_line_aeroway_idx on osm_line(aeroway);
-create index osm_line_waterway_idx on osm_line(waterway);
-create index osm_line_railway_idx on osm_line(railway);
-create index osm_line_bridge_idx on osm_line(bridge);
-create index osm_polygon_leisure_idx on osm_polygon(leisure);
-create index osm_polygon_aeroway_idx on osm_polygon(aeroway);
-create index osm_polygon_waterway_idx on osm_polygon(waterway);
-create index osm_polygon_natural_idx on osm_polygon("natural");
-create index osm_point_place_idx on osm_point(place);
-create index osm_line_zorder_idx on osm_line(z_order);
+create index OSM_PREFIX_polygon_building_idx on OSM_PREFIX_polygon(building);
+create index OSM_PREFIX_polygon_amenity_idx on OSM_PREFIX_polygon(amenity);
+create index OSM_PREFIX_polygon_landuse_idx on OSM_PREFIX_polygon(landuse);
+create index OSM_PREFIX_line_highway_idx on OSM_PREFIX_line(highway);
+create index OSM_PREFIX_line_aeroway_idx on OSM_PREFIX_line(aeroway);
+create index OSM_PREFIX_line_waterway_idx on OSM_PREFIX_line(waterway);
+create index OSM_PREFIX_line_railway_idx on OSM_PREFIX_line(railway);
+create index OSM_PREFIX_line_bridge_idx on OSM_PREFIX_line(bridge);
+create index OSM_PREFIX_polygon_leisure_idx on OSM_PREFIX_polygon(leisure);
+create index OSM_PREFIX_polygon_aeroway_idx on OSM_PREFIX_polygon(aeroway);
+create index OSM_PREFIX_polygon_waterway_idx on OSM_PREFIX_polygon(waterway);
+create index OSM_PREFIX_polygon_natural_idx on OSM_PREFIX_polygon("natural");
+create index OSM_PREFIX_point_place_idx on OSM_PREFIX_point(place);
+create index OSM_PREFIX_line_zorder_idx on OSM_PREFIX_line(z_order);
